build-release.sh

```diff
@@ -8,7 +8,7 @@ eval "$(go env)"
 VERSION=${1:-$(git describe --tags --always --dirty)}
 
 # To overwrite the version details, pass something as the second arg. Empty string disables it.
-VERSION_DETAILS=${2-"$(date --utc --iso-8601=s)/$(git describe --always --long --dirty)"}
+VERSION_DETAILS=${2-"$(date -u +"%Y-%m-%dT%H:%M:%S%z")/$(git describe --always --long --dirty)"}
 
```

Comment:
Does the MacOS support the `-I` flag, like alpine does? See how I changed this in the Dockerfile: https://github.com/loadimpact/k6/blob/5f12764462a93260eee80d88c749d30552e18749/Dockerfile#L5
If that works on mac it'd be best for them to be equivalent

Reply:
It does not, that's why I use a specific time format.

Reply:
Can you copy this to both the Dockerile and here then, so we use the exact same format in both places? Also according to https://stackoverflow.com/questions/7216358/date-command-on-os-x-doesnt-have-iso-8601-i-option , `date -u +%FT%T%z` is a shorter way to specify the same format and should work on both GNU and BSD `date`

Reply:
Fixed